### PR TITLE
Build template pages 

### PR DESCRIPTION
**Pull request by CodyGbengz**
**what does this PR do?**
create template pages for the projects front-end.
**Description of tasks to be completed**
-Create login/sign up page
-Create view top recipes page
-Create favorite recipes page
-Create a recipe details page
-Create a manage my recipes page
-Create a user profile page
**what are the relevant pivotal tracker stories?**
-#150576813
-#150576870
-#150576914
-#150576796
-#150576981

### CODE REVIEW

**Comment by CodyGbengz** [comment]
I've updated the templates based on the feedback I received on monday.  I'm yet to create modals for the add recipe page. Its just a form on a seperate page for now. Managing the limited time available is of the utmost importance. I'll implement too though.

**Comment by damidurodola** [comment]
@WillyWunderdog I noticed you've not implemented the feedback given concerning your indentation. I can still see some irregularities. Set your tabSize to use 2spaces in your editor.
The `search` functionality is not visible. Use a search icon for easy identification

File: templates/create recipe.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+          </div>
+          <div class="file-field input-field col s12 m12">
+            <h6><em><b>Ingredients</b></em></h6>
+            <small><em>Enter your Ingredients one at a time and hit enter</em></small>
+            <input id="first_name" placeholder="e.g 1 tsp dry pepper" type="text" class="validate">
+          </div>
+          <div class="input-field col s12 m12">
+            <h6><em><b>Directions</b></em></h5>
+              <small><em>Enter your directions one step at a time and hit enter</em></small>
+              <input id="first_name" placeholder="e.g Place all ingredients in a blender" type="text" class="validate">
```
Comment (damidurodola) [inline templates/create recipe.html]: A text-area will be appropriate here.

**Comment by CodyGbengz** [comment]
I've improved on the templates based on your last feedback. I'm yet to add an icon for the search bar though .

**Pull request merged**